src/uniref/types/entrySection.ts

```diff
@@ -21,7 +21,7 @@ export const getEntrySectionNameAndId = (
       };
     case EntrySection.Members:
       return {
-        name: 'Members',
+        name: 'members',
```

Comment:
if there's no way to conditionally add the 's' if more than 1 member, this should say "member(s)"

Reply:
Changed it to say member, and append "s" in the same bit where I prepended to count